src/entries/EntriesModel.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+require_once 'src/entries/Entry.php';
+
+// EntriesModel, to handle database queries for the entries table
+class EntriesModel
+{
+    public PDO $db;
+
+    public function __construct(PDO $db)
+    {
+        $this->db = $db;
+    }
+
+    public function getAllEntries()
+    {
+        $query = $this->db->prepare('
+        SELECT `id`, `title`, `description`
+        FROM `entries`;
+        ');
+        $query->execute();
+        $entries = $query->fetchAll();
+
+        $entryObjects = [];
+
+        foreach($entries as $entry) {
+            $entryObjects[] = new Entry($entry['id'], $entry['title'], $entry['description']);
+        }
+
+        return $entryObjects;
+    }
+
+    public function addNewEntry($db, string $title, string $description): bool 
```

Comment:
No need to have $db as a param here - because you already have it as a property in this class, you can just do `$this->db` when you need to prepare the query.

Reply:
Thanks, $db param removed, query prepare updated to use $this->db.